File: Revision_classes/05.Static AND non-static/Test1.java
class Example
{
	static int a = 10;
	static int b = 20;
	
	int x = 30;
	int y = 40;

    static void m1(){
		System.out.println("m1");
	}
    void m2(){
		System.out.println("m2");
	}
}
class Test1
{
	public static void main(String[] args) 
	{
		Example e1;
		e1 = new Example();
	//	e1.m2();
		
		Example e2;
		e2 = new Example();
	}
}

File: React/app7/src/Test1.js
export default function Test1() {
    const test = () => {
        console.log(1);
        console.log('before loop');
        for (var i = 1; i <= 9999999999; i++) {
        }
        console.log('after loop', i)
        console.log(2);
        console.log(3);
        console.log(4);
        console.log(5);
    }
    return (
        <>
            <button type="button" onClick={test}>test</button>
        </>
    );
}